src/main/java/com/matera/cadastrocentral/address/AddressController.java

```diff
@@ -23,8 +23,8 @@ Address getAddressById(@PathVariable UUID addressId) {
         return addressService.getAddressById(addressId);
     }
 
-    @GetMapping
-    List<Address> getAddressListByClientId(@RequestParam(value = "clientId", required = true) UUID clientId) {
+    @GetMapping("/client/{clientId}")
```

Comment:
I think at all paths we should pass the clientId, as Seabra did at his PR. Like "clients/{clientId}/addresses/{addressId}"